### PR TITLE
Nukeops leadership overhaul

### PR DESCRIPTION
**Pull request by N3X15**
# Executive Summary
Makes it clearer who has authority during nukeops, adds some warnings, and a macro for detecting leadership status.

**NOTE:** Does NOT include wardrobe changes.

# New Icons
image
Nukeop Leader HUD icon

# Changelog
:cl:
 - tweak: Nukeop leaders are a bit more obvious.  They now have a title in front of their name and a special HUD icon.
 - tweak: "Nukeops are now told to follow their leaders' instructions. Failure to comply may result in being stunned or even jobbans."

### CODE REVIEW

**Comment by N3X15** [comment]
Alternate icon?
image

**Comment by N3X15** [comment]
image

**Comment by Orangerush** [comment]
Our greatest ally

**Comment by ghost** [comment]
Gimme that sweet chevron baby

**Comment by ghost** [comment]
It should be L for loser

**Comment by ghost** [comment]
Why do you gotta make every PR about you?

**Comment by TheFriggenSawbones** [comment]
REMEMBER: You must follow your Leader's plans. Insubordination can lead to severe punishment, including a jobban from Nukeops.  
As Leader, you have sole authority in creating plans.  Stun and adminhelp (F1) if someone is being disobedient. 

you should probably talk to us admins before slipping that in, unless i'm missing something here.

**Comment by MusgardTiger** [comment]
Can you make it so the nukops leader starts with a handcuff so they can cuff the insubordinate scum on the spot to avoid trying to drag the guy and keep them stunned until they can find a handcuff box or something.

Also please add more details to the original PR post so people can see what is being added. (I meant it says "Nukeops are now told to follow their leaders' instructions." not "He can stun you and you might get into trouble for not listening to your leader")

**Comment by SonixApache** [comment]
in my experience; stunning armed individuals, especially in a group of like-minded armed individuals, usually results in chaos and collateral damage

**Comment by Zennyization** [comment]
I like this PR, it helps to resolve one of the most frustrating issues when playing Nukeops. When you are trying to make a plan at the start of the round but everybody has conflicting ideas and the chatbox just becomes a mess. Hopefully this means that there can atleast be some coherent conversation.

**Comment by Orangerush** [comment]
sawbones it says "may lead to jobban" you know if a nuke ops flukes so hard that it fucks up everyone else type deal (see boombox). Like I think how it's always been

**Comment by TheFriggenSawbones** [comment]
@Orangerush right but if that's what he's aiming for he needs to word it differently I think as we don't job ban people for not listening to the leaders, just fucking over all the nukies.

**Comment by Orangerush** [comment]
tbh not listening to the commander and fluking go hand in hand in my mind. Like if you're not going to listen to people in a team gamemode what are the chances of you fucking over everyone

**Comment by Shadowmech88** [comment]
The only way you could both not listen to your leader *and* not screw with his plan at all would be to just sit on the ship and do nothing, which I doubt is going to happen much.

**Comment by nuklearcellphoneg** [comment]
>okay we are going to be stealthy
>one guy goes loud fluke ops
>okay we are going loud
>one guy sneaks off fluke ops

Would be kind of nice if the leader could assign 'objectives' in the form of IC notes so that the nuke ops under them would know what's expected of them and admins later on could see whether they could be reasonably held accountable for deviating from the plan.

File: code/game/gamemodes/nuclear/nuclear.dm [hunk 1]
```diff
@@ -82,7 +82,7 @@
 							var/imageloc = synd_mind_1.current
 							if(istype(synd_mind_1.current.loc,/obj/mecha))
 								imageloc = synd_mind_1.current.loc
-							var/image/I = image('icons/mob/mob.dmi', loc = imageloc, icon_state = "synd")
+							var/image/I = image('icons/mob/mob.dmi', loc = imageloc, icon_state = (synd_mind_1 == ticker.mode.nukeop_leader) ? "synd-L" : "synd")
```
Comment (PJB3005) [inline code/game/gamemodes/nuclear/nuclear.dm:85]: Don't copy paste the nuke ops leader check around.

**Review by PJB3005 — CHANGES_REQUESTED** [review]

**Comment by Iamgoofball** [comment]
I like it, but I feel like nuke op leaders should have some way to penalize their guys if they do nothing. A leader needs to be able to stop their underlings from being idiots.

File: code/game/gamemodes/nuclear/nuclear.dm (hunk 1 above)
Comment (N3X15) [reply code/game/gamemodes/nuclear/nuclear.dm:85]: I don't understand the problem with this check.

**Comment by nuklearcellphoneg** [comment]
The possibility of OOC punishment is of course the penalty(after all, round ends nothing is going to happen whether they win or lose). And the sort of person who would literally do nothing at all... it's probably best they not play nuke ops. Or cult. Or (team game mode here).

Again, being able to give your underlings objectives via the Notes button would be good because it
would have what's expected written so the admins can properly hold the underling accountable as well as the leader, and whether or not they did what was expected of them. It would also be best to be able to assign these either to the whole squad or an individual.

Also it's likely best that, if this were added, it only be able to be done if the leader and the underling(s) were present at the home base, otherwise you have an OOC form of communication between the OPs.

File: code/game/gamemodes/nuclear/nuclear.dm (hunk 1 above)
Comment (PJB3005) [reply code/game/gamemodes/nuclear/nuclear.dm:85]: Make a macro or something to check whether a mind is the nukeops leader. Don't copy paste it.

**Comment by Iamgoofball** [comment]
Theoretically speaking, you shouldn't offload everything on admins when you can easily give nuke op leaders a way to 3 strike you're out system that does something like replacing a underling with a ghost or some shit.

**Comment by PJB3005** [comment]
That's way too open to abuse.

**Comment by Iamgoofball** [comment]
Yeah, it's just an example of something, though, and in the end banning someone for metagaming with ghost knowledge is a hell of a lot easier and simpler than arguing over whether the op was listening to the leader properly.

**Comment by nuklearcellphoneg** [comment]
Hence my suggesting to be able to assign Objectives in the form of notes(just to sidestep the cries of 'muh greentext' at round end); metagaming would not necessarily be easy to prove, it also opens the opportunity for a ghost who just died to the nuke ops to autistically rage out against the OPs that killed them(think in terms of when someone converts someone to cultist or a shade).

Rare sure but digressing, for example, how is the leader going to know the underling is disobeying in the first place? Goof, banning for meta vs proving they disobeyed instead of failed the objective is only easier if you also force the leader of the Nuke Ops to assume the 'computer guy talking to the guys in the field'(or if they don't want to do that, well, then admins must deal with a leader who strikes out an op with limited information, and that op would have the right to F1: THE LEADER STRUCK ME OUT 4NORAISIN).

Not that I'm against or for that, I think that kind of role sounds neat and plays into the Nuke OP helmet cameras, but  it's also the difference between the leader role being 'okay this is the plan LET'S DO IT' and somewhat putting them into a micromanaging position lest they risk being an ineffective leader.

**Comment by Iamgoofball** [comment]
are you high

**Comment by yclatious** [comment]
The genius of Nuke is simply not understood goofball, he is so high off his ass in a wooden cabin and generally insane he somehow makes sense.

**Comment by N3X15** [comment]
Or players could just follow the damn leader's instructions without fucking with a system currently being refactored.

**Comment by nuklearcellphoneg** [comment]
@Iamgoofball No, I'm Nuklear.

File: code/game/gamemodes/nuclear/nuclear.dm (hunk 1 above)
Comment (N3X15) [reply code/game/gamemodes/nuclear/nuclear.dm:85]: The other nukeop check macros work in the same fashion (using mobs rather than minds).

**Comment by DeityLink** [comment]
Neat

**Comment by N3X15** [comment]
What does this need?

**Comment by Shadowmech88** [comment]
Looks good to me.

**Pull request merged**